python/tvm/relay/qnn/op/qnn.py

```diff
@@ -180,15 +180,16 @@ def concatenate(data,
         The concatenated quantized tensor.
     """
 
-    data = list(data)
-    if not data:
-        raise ValueError("relay.concatenate requires data to be non-empty.")
+    if isinstance(data, (list, tuple)):
+        data = Tuple(data)
+    if isinstance(data, TupleWrapper):
```

Comment:
I think we should use `elif` so that there is no need to check again if `data` is list of tuple